fastlane/metadata/android/release_notes.xml

```diff
@@ -1,7 +1,35 @@
+<ar>
+2.18:
+* Redesigned signup flow
+</ar>
+<de-DE>
+2.18:
+* Redesigned signup flow
+</de-DE>
+<es-ES>
+2.18:
+* Redesigned signup flow
+</es-ES>
+<id>
+2.18:
+* Redesigned signup flow
+</id>
```

Comment:
@loremattei you were right in [this comment](https://github.com/Automattic/simplenote-android/pull/1348#discussion_r596697989). Some translations came back afterwards, but... they're all in English?! 🤔 

I thought it might have been a bug in out automation, but that's actually what's in the GlotPress source. E.g., [this](https://translate.wordpress.com/projects/simplenote/android/release-notes/ar/default/) for the Arabic:

![image](https://user-images.githubusercontent.com/1218433/112235877-9bfc0c80-8c93-11eb-8416-1d61a948f779.png)

Reply:
I was waiting to merge this for the translations, but given that, I'm going to merge and follow up elsewhere.